src/Copy/BrandCopy.elm

```diff
@@ -628,7 +628,7 @@ brandCopy key =
                 person to help me."""
 
         StoryThree2ImageAlt ->
-            CopyText "✨"
+            CopyText "Illustration of 4 persons sitting next to each other, hands folded, with their back to a woman with a phone next to her"
```

Comment:
Consider something like "Illustration of 4 persons sitting next to each other, arms folded, with their backs to a woman sitting with a phone and her hands in the air like she's given up"